components/layout/index.styl

```diff
@@ -1,12 +1,13 @@
-@import '../../node_modules/nib'
+// Common (includes Nib)
+@import '../../node_modules/artsy-elan/components/lib'
+
+// Components
+@import '../../node_modules/artsy-elan/assets/stylesheets/components'
+@import '../nav'
 @import '../header'
+
 @import '../footer'
 @import '../body'
-@import '../carousel/carousel.styl'
-@import '../carousel/desktop_carousel.styl'
-@import 'button'
-@import '../header/desktop'
+@import '../carousel/carousel'
+@import '../carousel/desktop_carousel'
```

Comment:
I think these desktop specific stylesheets are a bit weird/overkill

It makes sense to me to write the "desktop" version as the default then nest a media query or two around for anything that's responsive